package-examples/subpackages/wordpress/README.md

```diff
@@ -0,0 +1,92 @@
+  Here are the steps to get, view, set and apply the package contents
+  
+  ### Fetch a remote package
+  Get the example package on to local using `kpt pkg get`
+  
+  $ kpt pkg get https://github.com/GoogleContainerTools/kpt.git/package-examples/subpackages/wordpress \
+    wordpress
+  
+      fetching package /package-examples/subpackages/wordpress from https://github.com/GoogleContainerTools/kpt to wordpress
+  
+  ### View the package contents
+  List the package contents in a tree structure.
+  
+  $ kpt cfg tree wordpress/
+  
+      wordpress
+      ├── [Kptfile]  Kptfile wordpress
+      ├── [wordpress-deployment.yaml]  Deployment wordpress
+      ├── [wordpress-deployment.yaml]  Service wordpress
+      ├── [wordpress-deployment.yaml]  PersistentVolumeClaim wp-pv-claim
+      └── Pkg: mysql
+          ├── [Kptfile]  Kptfile mysql
+          ├── [mysql-deployment.yaml]  PersistentVolumeClaim mysql-pv-claim
+          ├── [mysql-deployment.yaml]  Deployment wordpress-mysql
+          └── [mysql-deployment.yaml]  Service wordpress-mysql
+  
+  The fetched package contains setter parameters which can be used to set configuration
+  values from the commandline, list them
+  
+  $ kpt cfg list-setters wordpress/
+  
+      wordpress/
+               NAME             VALUE      SET BY   DESCRIPTION   COUNT   REQUIRED  
+        gcloud.core.project   PROJECT_ID                          3       No        
+        image                 wordpress                           1       No        
+        tag                   4.8                                 1       No        
+        teamname              YOURTEAM                            3       Yes       
+      
+      wordpress/mysql/
+               NAME             VALUE      SET BY   DESCRIPTION   COUNT   REQUIRED  
+        gcloud.core.project   PROJECT_ID                          3       No        
+        image                 mysql                               1       No        
+        tag                   5.6                                 1       No        
+        teamname              YOURTEAM                            3       Yes       
+        
+   If you have `gcloud` set up on your local, you can observe that the value of the setter
+   `gcloud.core.project` is set automatically when the package is fetched.  
+   `gcloud` config setters are automatically set deriving the values from the output of
+   `gcloud config list` command, when the package is fetched using [kpt pkg get].
+  
+  ### Provide the setter values
+  `set` operation modifies the resource configuration in place by reading the resources,
+  changing parameter values, and writing them back.
+  
+  In `list-setters` output, `namespace` setter is marked as required by the package
+  publisher, hence it is mandatory to set it to a new value. You may set other setter
+  parameters, either selectively in few of the packages or in all the them.
```

Comment:
sorry for being a stickler here: Are we just referring to 'setter parameters' everywhere? I think you want to reduce as many redundant words and concepts as possible. So:

- In one place (in setters documentation) state that setters are how kpt implements parameterization.
- In rest of kpt docs, just refer to it as 'setters'.

Phrasing to be used consistency everywhere:

A producer "declares a setter definition"
A consumer "provides a setter value". Currently this is imperative, but in the future it will be declarative: "Declare setter values".

Make the changes as a follow up. Don't block this PR though.

Reply:
Agree. Simplified all the text and added references rather than redefining the concepts. Thanks.